core/bones/key.py

```diff
@@ -116,14 +120,13 @@ def serialize(self, skel: 'SkeletonInstance', name: str, parentIndexed: bool) ->
             return True
         return False
 
-
     def buildDBFilter(
         self,
-          name: str,
-          skel: 'viur.core.skeleton.SkeletonInstance',
-          dbFilter: db.Query,
-          rawFilter: Dict,
-          prefix: Optional[str] = None
+        name: str,
+        skel: 'viur.core.skeleton.SkeletonInstance',
+        dbFilter: db.Query,
+        rawFilter: Dict,
+        prefix: Optional[str] = None
```

Comment:
Is it necessary to provide full type annotations in sub-classes as well?
I would like to keep maintenance overhead low, if possible.